Core/Inc/canardRxQueue.h

```diff
@@ -0,0 +1,18 @@
+#ifndef CANARD_RX_QUEUE_H
+#define CANARD_RX_QUEUE_H
+
+#include <stdlib.h>
+#include <canard.h>
+
+struct CanardRxQueueItem {
+    struct CanardRxQueueItem* next;
+    CanardCANFrame frame;
+};
+
+extern struct CanardRxQueueItem* rxQueueHead;
+extern struct CanardRxQueueItem* rxQueueTail;
+
+void enqueueRxFrame(CanardCANFrame* rx_frame);
```

Comment:
more comments to describe what the function does, its parameters, and return value are greatly appreciated.